src/algebra/big_operators.lean

```diff
@@ -4,7 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
 -/
 
-import data.finset
+import data.finset.intervals
+import data.finset.fold
+import data.finset.powerset
+import data.finset.pi
```

Comment:
Would it make sense to create `data/finset/default.lean` instead? (Similarly for `data/multiset/default.lean`.)

Reply:
Except that part of the point of this is to thin out import trees. (Admittedly this is a smaller point than trying to make the files digestible.) My preference would be to leave as is.

Reply:
Actually, sorry --- my preference would be to add the `default.lean` files as you suggest, but then not to use them. (But so they are readily available for anyone who doesn't want to be bothered working out which imports they need.)

Reply:
OK, I don't have strong feelings on whether we should use them in mathlib, so I think we're in agreement.

Reply:
Done.